### PR TITLE
Delete strange use of several in readme.md

### PR DESCRIPTION
**Pull request by CrafterKolyan**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Activate environment:
 ```
 source env/bin/activate
 ```
-Run several N - 1 clients by:
+Run N - 1 clients by:
```
Comment (nikitabobko) [inline readme.md]: Thank you. It must be "Run first N-1 clients by:"

**Review by nikitabobko — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (nikitabobko) [reply readme.md]: Do you want to fix your pull request or do I need to fix it myself in `master`?

Comment (CrafterKolyan) [reply readme.md]: Fixed, but choose "squash and merge" when will be merging

Comment (CrafterKolyan) [reply readme.md]: Also as I know you can edit pull request by yourself

**Pull request merged**